build.sbt

```diff
@@ -12,7 +12,6 @@ val akkaV = "2.6.18"
 val akkaHttpV = "10.2.7"
 val slickV = "3.3.3"
 val workbenchGoogleV = "0.21-31be16e8-SNAP"
-val rawlsModelV = "0.1-384ab501b"
```

Comment:
I got rid of the rawls-model junk too. Much cleaner to just use the existing workbench-libs implementations for this.

I think Orch still uses some rawls-model stuff, but it'd be nice to deprecate it eventually.